src/app/pages/Login/Login.tsx

```diff
@@ -1,17 +1,29 @@
 import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { User } from '../../../types';
+import { postLoginUser } from '../../../utils/api';
 import Button from '../../components/Button/Button';
 import PasswordIcon from '../../components/Icons/PasswordIcon';
 import UserIcon from '../../components/Icons/UserIcon';
 import LabeledInput from '../../components/LabeledInput/LabeledInput';
 import styles from './Login.module.css';
 
 function Login(): JSX.Element {
-  const [username, setUsername] = useState('');
-  const [password, setPassword] = useState('');
+  const history = useHistory();
+  const [username, checkUsername] = useState('');
+  const [password, checkPassword] = useState('');
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
 
-  function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+  async function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
     event.preventDefault();
-    alert(`Login with ${username}:${password}`);
+
+    try {
+      const user: Partial<User> = { username, password };
+      await postLoginUser(user);
+      history.push('/register');
```

Comment:
Why should you be redirected to `/register` after a successful login?

Reply:
Thanks! That was not planned.